integration_tests/iceberg-sink/README.md

```diff
@@ -34,3 +34,45 @@ docker compose exec presto presto-cli --server localhost:8080
 ```sql
 select user_id, count(*) from iceberg.demo_db.demo_table group by user_id
 ```
+
+## Demo: Iceberg Compaction With Airflow
```

Comment:
Could you describe the setting of compaction in this file? Such as the airflow task frequency, and the parameter of the compaction task.